### ISSUE TITLE
No response when pressing the "SET" button in PTZ control

### ISSUE THREAD

**Issue by IgorA100** [ZoneMinder/zoneminder#3890]
- 1.37.56~20240322163032

Button press
`<button type="button" class="ptzTextBtn" value="Set" data-toggle="modal" data-target="#ctrlPresetModal"><?php echo translate('Set') ?></button>` 
skins\classic\includes\control_functions.php not processed in \skins\classic\views\js\watch.js

https://forums.zoneminder.com/viewtopic.php?p=133906#p133906

**Comment by IgorA100** [comment]
data.html is empty, strange
```
function getCtrlPresetModal() {
  $j.getJSON(monitorUrl + '?request=modal&modal=controlpreset&mid=' + monitorId)
      .done(function(data) {
        insertModalHtml('ctrlPresetModal', data.html);
```

**Comment by IgorA100** [comment]
The problem is here \ajax\modals\controlpreset.php:
if (!canEdit('Monitors')) return ' ';

**Comment by connortechnology** [comment]
In a multi-server setup, we must pass authentication info along which we weren't doing.  Fixed by dd0b2bcf91a9be6537f311aff4a61b2b2556c7cf
